math-plugin/src/test/java/com/github/underscore/math/MathTest.java

```diff
@@ -264,8 +265,12 @@ public void chain() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new String[] {""}).filter(new Predicate<String>() {
             public Boolean apply(String str) { return true; } });
+        $.chain(new String[] {""}).filterIndexed(new PredicateIndexed<String>() {
+            public boolean apply(int index, String str) { return true; } });
         $.chain(new String[] {""}).reject(new Predicate<String>() {
             public Boolean apply(String str) { return true; } });
+        $.chain(new String[] {""}).rejectIndexed(new PredicateIndexed<String>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.rejectIndexed\(PredicateIndexed\) ignored, is this OK in com\.github\.underscore\.math\.MathTest\.chain\(\)